src/components/ebay-toggle-button-group/examples/icons.marko

```diff
@@ -0,0 +1,32 @@
+<ebay-toggle-button-group columns=3 layoutType="list" ...input>
+    <@toggle-button title="Icon 1" subtitle="Car">
```

Comment:
We should change it there. 
For now lets keep title support so its not breaking. Just do a check if title is present render with that otherwise renderBody
And remove title from docs. 
Just one more question. Should subtitle be an attribute tag? Could there be anything in subtitle other than text?